### PR TITLE
doc: Add instructions on obtaining the Self-hosted version CY-3783

### PR DESCRIPTION
**Pull request by pauloribeiro-codacy**
Adds brief instructions on how to see the version of the Codacy Self-hosted instance to the two pages where knowing the version is more relevant:

image

---

image

### CODE REVIEW

**Review by inesgomas — APPROVED** [review]

File: docs/assets/includes/self-hosted-version.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+!!! tip
+    To see the version of your Codacy Self-hosted instance click your avatar on the top right-hand corner of any Codacy page: 
+
+    ![Obtaining the Codacy Self-hosted version](./images/self-hosted-version.png)
```
Comment (pauloribeiro-codacy) [inline docs/assets/includes/self-hosted-version.txt:4]: I decided to include the instructions on how to see the version in a "tip" admonition card in a centralized file that is then included in the pages where this information is useful.

**Comment by pauloribeiro-codacy** [comment]
@inesgomas I saw the updated menu on the dev environment and already updated the screenshot. :heavy_check_mark: From what I could see in your changes to the code, the version string will be displayed in the format `vM.m.p`, but please check.

**Review by inesgomas — APPROVED** [review]

**Review by inesgomas — COMMENTED** [review]
Yes, that will be the format, nice job 👍 

But this will only be available on next self-hosted release

**Comment by pauloribeiro-codacy** [comment]
> Yes, that will be the format, nice job 
> 
> But this will only be available on next self-hosted release

Yes, no worries - the self-hosted documentation is typically only updated when there's a new Self-hosted release. Or if it is updated before, it will be only under the "Cloud (Latest)" documentation version. :+1: 

image

**Pull request merged**